test/functional/api/v1/deletions_controller_test.rb

```diff
@@ -108,6 +108,29 @@ class Api::V1::DeletionsControllerTest < ActionController::TestCase
         end
       end
 
+      context "with api key gem scoped" do
+        context "to another gem" do
+          setup do
+            ownership = create(:ownership, user: @user, rubygem: create(:rubygem, name: "test"))
+            @api_key.update(rubygem: ownership.rubygem)
```

Comment:
This setup can lot more clearer if make use new API key with explicit names to reflect intention.